lib/src/location_options.dart

```diff
@@ -24,7 +24,8 @@ class LocationOptions extends LayerOptions {
       this.onLocationRequested,
       @required this.buttonBuilder,
       this.markerBuilder,
-      this.updateIntervalMs = 1000})
+      this.updateIntervalMs = 1000,
+      this.initiallyRequest = false})
```

Comment:
Can you switch the default to `true` so that the behavior stays by default the same as before.